### PR TITLE
Add webhook for all CRs

### PR DESCRIPTION
**Pull request by kstrenkova**
There is a need to add webhooks to all CRs like it was done for Tempest CR. In this patch webhooks for Tobiko, HorizonTest and AnsibleTest are added. There is an effort to make the webhook as generic as possible to minimize the duplicate code.

### CODE REVIEW

**Comment by lpiwowar** [comment]
Can we split this please into two PRs, please?
* One for addition of the webhooks
* Second for merging of the workflows

**Review by lpiwowar — CHANGES_REQUESTED** [review]
Can we split this please into two PRs, please?
* One for addition of the webhooks
* Second for merging of the workflows

**Review by lpiwowar — COMMENTED** [review]
Looks good to me there are just two tasks left one for @kstrenkova and second for @lpiwowar (I'm going to take a look on this on Monday). Otherwise the PR looks good to me. These are just small things to polish:).

File: hack/configure_local_webhook.sh [hunk 1]
```diff
@@ -1,6 +1,22 @@
 #!/bin/bash
 set -ex
 
+if [ $# -eq 0 ]; then
+    echo "Error: No arguments provided."
+    echo "You need to specify the name of the resource in lowercase, e.g. tempest"
+    exit 1
+fi
```
[the remark this answers is not in the document]
Comment (kstrenkova) [reply hack/configure_local_webhook.sh]: Ack, I changed the patch accordingly.

**Pull request merged**